packages/mobile/src/components/audio/GoogleCast.tsx

```diff
@@ -1,5 +1,6 @@
 import { useCallback, useEffect, useState } from 'react'
```

Comment:
Don't suppose you tested the google cast functionality? I know we're using the same patterns here as elsewhere, just have a nagging fear that there will be something weird here.

Reply:
i think it may be _already_ broken, but yes, will test on staging and probably end up making a better ticket